proposals/0344-distributed-actor-runtime.md

```diff
@@ -895,20 +894,20 @@ public protocol DistributedTargetInvocationEncoder {
   mutating func recordGenericSubstitution<T>(_ type: T.Type) throws
 
   /// Record an argument of `Argument` type in this arguments storage.
-  ///
-  /// Ad-hoc requirement.
+  //
+  // Ad-hoc requirement.
```

Comment:
Why `//` instead of `///` for ad-hoc requirement?

Reply:
This follows the style of L288-289.
Do you think `///` is better?

Reply:
Yeah, I think they all need to be `///`

Reply:
They were // because that's a note to the reviews, not actual doc, but either is fine